### PR TITLE
TSK-1597: Unified HTTP Request in Rest Tests

### PR DESCRIPTION
**Pull request by tge20**
---
 
### For the submitter:
- [ ] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
- [x] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
- [ ] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
- [ ] I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I put the ticket in review
- [ ] After integration of the pull request, I verified our [bluemix test environment](http://taskana.mybluemix.net/taskana) is not broken

### Verified by the reviewer:
- [ ] Commit message format → TSK-XXX: Your commit message.
- [ ] Submitter's update to [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) is sufficient
- [ ] SonarCloud analysis meets our standards
- [x] Update of the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana) reflects changes
- [x] PR fulfills the ticket
- [x] Edge cases and unwanted side effects are tested
- [x] Readability

### CODE REVIEW

File: rest/taskana-rest-spring/src/test/java/pro/taskana/task/rest/TaskControllerIntTest.java [hunk 1]
```diff
@@ -79,283 +75,243 @@
...
+          ThrowingCallable httpCall =
+              () -> {
+                TEMPLATE.exchange(url, HttpMethod.POST, auth, TASK_MODEL_TYPE);
+              };
+          assertThatThrownBy(httpCall)
+              .isInstanceOf(HttpClientErrorException.class)
+              .hasMessageContaining("Format of custom attributes is not valid")
+              .extracting(ex -> ((HttpClientErrorException) ex).getStatusCode())
+              .isEqualTo(HttpStatus.BAD_REQUEST);
+        };
```
Comment (mustaphazorgati) [inline rest/taskana-rest-spring/src/test/java/pro/taskana/task/rest/TaskControllerIntTest.java:268]: :clap:

**Review by mustaphazorgati — APPROVED** [review]
